tests/www/test_views.py

```diff
@@ -569,6 +569,11 @@ def test_code(self):
         resp = self.client.get(url, follow_redirects=True)
         self.check_content_in_response('example_bash_operator', resp)
 
+    def test_code_no_file(self):
+        url = 'code?dag_id=example_bash_operator_xxx'
+        resp = self.client.get(url, follow_redirects=True)
```

Comment:
This error is dag not found (i.e. it's not even in the database) which is very different to dag _source_ code could not be read by webserver. i.e. this test isn't checking what you describe in the PR.

(This request should result in a 302 back to home and show a flash of "DAG example_bash_operator_xxx not found"  or similar)

Reply:
Yes, sorry, I was being lazy. I did not know how to write the test I needed so I wrote this test instead. I will try to figure it out.